curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -354,9 +357,10 @@ impl<C: Command, CE: CommandExecutor<C>> Filter<C, CE> {
                     }
                     false
                 }
-                (ExeState::Executed(_), AsState::AfterSynced) => true,
-                (ExeState::Executing | ExeState::Executed(_), AsState::NotSynced(_))
-                | (ExeState::Executing, AsState::AfterSyncReady(_, _) | AsState::AfterSyncing)
+                (ExeState::Executed(false), AsState::AfterSyncReady(_, _))
+                | (ExeState::Executed(_), AsState::AfterSynced) => true,
+                (ExeState::Executing(_) | ExeState::Executed(_), AsState::NotSynced(_))
```

Comment:
If it's `(ExeState::Executed(false), AsState::NotSynced(_))`, it should be able to return true?

Reply:
`(ExeState::Executed(false), AsState::NotSynced(_))` will transform to `(ExeState::Executed(false), AsState:: AfterSyncReady(_))` and finally remove.